spec/features/sales_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.feature "UserAddItemsToCarts", type: :feature do
+
+  describe "add food to sale" do
+    scenario "admin add a food to sales" do
+      food = create(:food)
+      admin = create(:admin)
+      visit root_path
+      click_link('Log in')
+      fill_in "session_email", with: 'admin@andela.com'
+      fill_in "session_password", with: 'andela'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.